docs/querying/basics.md

```diff
@@ -204,6 +233,10 @@ The same works for range vectors. This returns the 5-minute rate that
 
     rate(http_requests_total[5m] offset 1w)
 
+When querying for samples in the past, a negative offset will enable temporal comparisons forward in time:
```

Comment:
This is not part of this PR.

Reply:
Yup, this is my error.  I did a cp and didn't remove this update.